packages/components/src/internal/components/editable/models.ts

```diff
@@ -129,10 +130,11 @@ export class EditorModel
     declare focusValue: List<ValueDescriptor>;
     declare id: string;
     declare isPasting: boolean;
+    declare isSparseSelection: boolean;
```

Comment:
maybe add a comment about where this is set (i.e. `applyEditableGridChangesToModels`)

Reply:
Done.